src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -296,13 +296,10 @@ const ReimbursementRequestDetailsView: React.FC<ReimbursementRequestDetailsViewP
         isReimbursementRequestDenied(reimbursementRequest)
     },
     {
-      title: 'Approve',
+      title: isReimbursementRequestSaboSubmitted(reimbursementRequest) ? 'Sabo Info' : 'Submit to Sabo',
       onClick: () => setShowSubmitToSaboModal(true),
       icon: <CheckIcon />,
```

Comment:
can you change this icon to a clipboard icon if it's already been submitted to sabo

Reply:
where do I get a clipboard icon? i couldn't find it anywhere else in the code